static/js/github-create-branch.js

```diff
@@ -106,7 +106,7 @@ const loadBranches = () => {
           }
         }
       });
-      $("#ghParentBranch").select2("val", data.repository.defaultBranchRef.name);
+      $("#ghParentBranch").select2("val", data[0].name);
```

Comment:
We need to select default branch.

Reply:
Unfortunately this REST API simply returns the list of the branch and there is no attribute to recognize the default branch.

Reply:
For now, I've added another Request to fetch the default branch for the selected Repo